evalml/utils/gen_utils.py

```diff
@@ -300,9 +300,11 @@ def _convert_to_woodwork_structure(data):
         ww_data = pd.DataFrame(ww_data)
 
     # Convert pandas data structures to Woodwork data structures
+    ww_data = ww_data.copy()
     if isinstance(ww_data, pd.Series):
         return ww.DataColumn(ww_data)
-    return ww.DataTable(ww_data, copy_dataframe=True)
+
+    return ww.DataTable(ww_data)
```

Comment:
Hmm, so is `DataTable` copying implicitly now? The [docs for the constructor](https://woodwork.alteryx.com/en/stable/generated/woodwork.datatable.DataTable.html#woodwork.datatable.DataTable) just say "create datatable"

Reply:
Yup, see description for https://github.com/Featurelabs/woodwork/pull/398!

Reply:
Thanks @angela97lin ! Filed https://github.com/FeatureLabs/woodwork/issues/420